File: epi_judge_java/epi/SumRootToLeaf.java
package epi;
import epi.test_framework.EpiTest;
import epi.test_framework.GenericTest;
public class SumRootToLeaf {
  @EpiTest(testDataFile = "sum_root_to_leaf.tsv")

  public static int sumRootToLeaf(BinaryTreeNode<Integer> tree) {
    // TODO - you fill in here.
    return 0;
  }

  public static void main(String[] args) {
    System.exit(
        GenericTest
            .runFromAnnotations(args, "SumRootToLeaf.java",
                                new Object() {}.getClass().getEnclosingClass())
            .ordinal());
  }
}

File: epi_judge_cpp/sum_root_to_leaf.cc
#include "binary_tree_node.h"

int SumRootToLeaf(const unique_ptr<BinaryTreeNode<int>>& tree) {
  // Implement this placeholder.
  return 0;
}

#include "test_framework/generic_test.h"

int main(int argc, char* argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::vector<std::string> param_names{"tree"};
  GenericTestMain(args, "sum_root_to_leaf.tsv", &SumRootToLeaf,
                  DefaultComparator{}, param_names);
  return 0;
}
